zulipterminal/ui_tools/boxes.py

```diff
@@ -64,7 +65,8 @@ def stream_box_view(self, caption: str='', title: str='') -> None:
         self.msg_write_box = ReadlineEdit(multiline=True)
         self.msg_write_box.enable_autocomplete(
             func=self.generic_autocomplete,
-            key=keys_for_command('AUTOCOMPLETE').pop()
+            key=keys_for_command('AUTOCOMPLETE').pop(),
+            key_reverse=keys_for_command('AUTOCOMPLETE_REVERSE').pop()
```

Comment:
@neiljp I didn't fully understand you here, could you please elaborate?
You mean, we should restrict commands like autocomplete to have only a single key?

Reply:
To clarify, I'm not suggesting that this is necessary to include in this PR, but it seems like a an issue to address at some point.

If the urwid-readline can only manage one key for (each) autocomplete, then it seems it would be advisable for us to have a way to verify that these entries in `keys.py` (and potentially others) have only one entry, and maybe give a warning. Otherwise if the project or users add keys to these entries, they may wonder why only one key works.

One alternative would be for urwid-readline to have the potential to accept multiple keys for autocomplete, of course.